Silicon/TigerlakePchPkg/FitTbl/FitTbl.inf

```diff
@@ -0,0 +1,31 @@
+## @file
+#  Provide FSP information header
+#
+# Copyright (c) 2006 - 2016, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent

```

Comment:
do you need FitTabl.aslc and FitTbl.inf at all?
FIT table are dynamically generated by SBL build.  It should not be required.

Reply:
yes. Please remove FitTbl from this package.

Reply:
Done removed.